### PR TITLE
Update onboarding-macos.png

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the screenshot

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7510

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md [hunk 1]
```diff
@@ -46,13 +46,13 @@ You'll need to take the following steps:
 
 ## Step 1: Get the Microsoft Defender for Endpoint onboarding package
 
-1. In [Microsoft 365 Defender](https://security.microsoft.com), navigate to **Settings > Onboarding**.
+1. In [Microsoft 365 Defender](https://security.microsoft.com), navigate to **Settings >Endpoints> Onboarding**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-jamfpro-policies.md]:
```suggestion
1. In [Microsoft 365 Defender](https://security.microsoft.com), navigate to **Settings > Endpoints > Onboarding**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**